spec/models/news_item_spec.rb

```diff
@@ -1,5 +1,67 @@
-require 'rails_helper'
+require "rails_helper"
 
 RSpec.describe NewsItem, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "validation" do
+    it { should validate_presence_of(:title) }
```

Comment:
A heads up that you can use the `expect` syntax for shoulda methods. Keeps the expectations uniformed.

```ruby
it { is_expected.to validate_presence_of(:title) }
```